data/time_series.go

```diff
@@ -30,6 +32,21 @@ const (
 	TimeSeriesTypeWide
 )
 
+type FillMode string
+
+const (
+	FILL_MODE_PREVIOUS FillMode = "previous"
```

Comment:
Everywhere else for enums this would be FillModePrevious, also being doing iota enums for this sort of thing, so should probably stay consistent.